package-lockfile.el

```diff
@@ -0,0 +1,7 @@
+(;; Our direct dependencies
+ ("magit" . "v4.0.0")
+ ("transient" . "v0.8.0")
+ ("with-editor" . "v3.4.1")
+ ;; Transitive
+ ("dash" . "2.19.1"))
+:gamma
```

Comment:
Melpa would include this in the package, so you would have to edit the Melpa recipe to exclude it again. Can the value be provided directly in `straight-profiles` or does that insist on a file? Is this only used during testing, and if so, might it not make more sense to place it in `tests/` or `.github/workflows/`? If located there, the recipe would not have to be adjusted. If you keep it at the top-level, consider making it a hidden file (the recipe would have to be adjusted anyway [though that's something I might want to change in Melpa/`package-build.el`, after careful consideration]).

Reply:
It's test-only, I'll move it there.